apps/velo-external-db/test/storage/schema_provider.spec.ts

```diff
@@ -87,7 +87,7 @@ describe(`Schema API: ${currentDbImplementationName()}`, () => {
     test('add column on a an existing collection', async() => {
         await env.schemaProvider.create(ctx.collectionName, [])
         await env.schemaProvider.addColumn(ctx.collectionName, { name: ctx.columnName, type: 'datetime', subtype: 'timestamp' })
-        await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual(collectionToContainFields(ctx.collectionName, [{ field: ctx.columnName, type: 'datetime' }], env.capabilities))
+        await expect( env.schemaProvider.describeCollection(ctx.collectionName) ).resolves.toEqual(collectionToContainFields(ctx.collectionName, [{ field: ctx.columnName }], env.capabilities))
```

Comment:
I know that google sheets support only string, but we don't want to check for all the other implementations that it created with the right type?

Reply:
Any column that is not `_createdDate` or  `_updateDate` is a text type column, therefore, I have no way of knowing that the column created in the test is of date type.
In addition, in the past we did not check the column type, I added it a few PRs ago, and now it's coming back to bite me 😞